src/vsTTempSmooth_AVX2.cpp

```diff
@@ -1,6 +1,148 @@
 #include "VCL2/vectorclass.h"
 #include "vsTTempSmooth.h"
 
+short my_extract_epi16_from256(__m256i ymm_data, int idx)
```

Comment:
Can you put the new functions (non-VCL2) in an own file (something like vsTTempSmooth_pmod1_AVX2.cpp)?

Reply:
This function was used in semi-SIMD old design and not used in current full-SIMD design of processing function in the _AVX2.cpp file. It can be removed if no more needed for possible new designs and debugs.

Reply:
I don't meant that specific function but all the new AVX2 additions. That way the VCL code will be separated from the intrinsics code.